packages/gatsby-theme-carbon/src/components/DoDontExample/do-dont-example.scss

```diff
@@ -100,7 +100,7 @@
 }
 
 .#{$prefix}--example--correct .#{$prefix}--example-card::before {
-  background: $support-02;
+  background: #24a148;
```

Comment:
Is there a reason we are hard-coding the colors instead of using a token?

Reply:
Not sure, this was the color requested in the issue, the token was displaying the wrong color. 
This component is also being deprecated in favor of the newer one, so I didn't spend any time trying to figure out the token issue.